tests/pw/feature-map/feature-map.yml

```
@@ -110,8 +112,14 @@
           vendor can export products: true
           vendor can duplicate product: true
           vendor can permanently delete product [lite]: true
+          vendor can reset filter [lite]: true
```

yamllint

[error] duplication of key "vendor can reset filter [lite]" in mapping

(key-duplicates)